torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -122,7 +141,7 @@ def _lower_to_native_backend(model: QuantizedGraphModule) -> QuantizedGraphModul
     to the native backend in PyTorch (fbgemm/qnnpack), both backends shares the same
     operator signature so they can be lowered with the same function
     """
-    for ref_class in LOWER_MODULE_MAP.keys():
+    for ref_class in list(LOWER_MODULE_MAP.keys()) + list(LOWER_FUSED_MODULE_MAP):
```

Comment:
do you need to call `.keys()` on LOWER_FUSED_MODULE_MAP as well?

Reply:
Oh yes, `LOWER_FUSED_MODULE_MAP` used to be a set so I forgot to change this one too